### DISCUSSION TITLE
FAILURE: Build failed with an exception.

### DISCUSSION THREAD

**Discussion by Shkiper228** [SKCraft/Launcher#516] [I need help!] [answered]
After running the build.bat file, it throws this

* What went wrong:
Could not determine the dependencies of task ':launcher-fancy:shadowJar'.
> Could not resolve all dependencies for configuration ':launcher-fancy:runtimeClasspath'.
   > Could not resolve io.github.cottonmc.insubstantial:substance:7.3.1-SNAPSHOT.
     Required by:
         project :launcher-fancy
      > Could not resolve io.github.cottonmc.insubstantial:substance:7.3.1-SNAPSHOT.
         > Unable to load Maven meta-data from https://maven.offbeatwit.ch/repository/snapshots/io/github/cottonmc/insubstantial/substance/7.3.1-SNAPSHOT/maven-metadata.xml.
            > Could not GET 'https://maven.offbeatwit.ch/repository/snapshots/io/github/cottonmc/insubstantial/substance/7.3.1-SNAPSHOT/maven-metadata.xml'.
               > Connect to maven.offbeatwit.ch:443 [maven.offbeatwit.ch/163.172.169.69] failed: Connection timed out: connect

**Comment by hedgehog1029** [comment, marked as answer]
That package is definitely published; try again?

If not - seems like you might not be able to reach my maven server? Are you on a network with a restrictive firewall (e.g. corporate or education)?

**Comment by Shkiper228** [comment]
No, I'm on a public network. I have my firewall turned off
I'm not very good at networks. So, could you please explain what this error means?

**Comment by Shkiper228** [comment]
Looks like I found a potential solution

I'm from Ukraine, but without vpn the site 
https://maven.offbeatwit.ch/repository/snapshots/io/github/cottonmc/insubstantial/substance/7.3.1-SNAPSHOT/maven-metadata.xml does not respond

**Reply by hedgehog1029** [reply]
Oh, interesting - the server's in France, not sure why it'd be blocked specifically.

I could try putting it behind Cloudflare...

**Comment by Shkiper228** [comment]
I found a solution
For those who have such a problem - just use vpn!
